src/core/lang.h

```diff
@@ -29,7 +29,8 @@ typedef enum {
     MESSAGE_TYPE_TUTORIAL = 4,
     MESSAGE_TYPE_TRADE_CHANGE = 5,
     MESSAGE_TYPE_PRICE_CHANGE = 6,
-    MESSAGE_TYPE_INVASION = 7
+    MESSAGE_TYPE_INVASION = 7,
+    MESSAGE_TYPE_EVENT = 8
```

Comment:
Feel free to pushback on new type of message. To me, this looks like the most extensible and generic approach.

Reply:
Agreed.